packages/agents/sequencer/src/lib/operations/auctions.ts

```diff
@@ -131,81 +100,112 @@ export const selectBids = async (_requestContext: RequestContext) => {
       const auction = await cache.auctions.getAuction(transferId);
       if (auction) {
         const startTime = Number(auction.timestamp);
-        const elapsed = getNtpTimeSeconds() - startTime;
+        const elapsed = (getNtpTimeSeconds() - startTime) * 1000;
         if (elapsed > AUCTION_PERIOD) {
           auctions[transferId] = auction;
         }
       }
     }),
   );
 
-  for (const transferId of Object.keys(auctions)) {
-    const { bids, origin, destination } = auctions[transferId];
-
-    // TODO: deprecate... necessary for now
-    const bidData = await cache.auctions.getBidData(transferId);
-    if (!bidData) {
-      logger.error("Bid data not found for transfer!", requestContext, methodContext, undefined, {
-        transferId,
+  await Promise.all(
```

Comment:
concerned about doing a promise.all here... eventually, we need to account for the fact that one transfer's auction can affect another.

A router might have $100, and bid on 2 $100 transfers. If we execute both of them simultaneously, it might seem like they pass at first... but only one can go through.

So we have to do a for loop, at the very least by domain.

Should update our local tracking of routers' funds in the cache or something when we spend a router's liq on a transfer.